SwiftVideoBackground/Sources/VideoBackground.swift

```diff
@@ -96,131 +99,126 @@ public class VideoBackground {
+        guard let path = Bundle.main.path(forResource: name, ofType: type) else {
+            throw VideoBackgroundError.videoNotFound(VideoInfo(name: name, type: type))
+        }
+        let url = URL(fileURLWithPath: path)
+        play(
             view: view,
-            videoInfos: [VideoInfo(name: name, type: type)],
+            url: url,
             isMuted: isMuted,
             alpha: alpha,
-            willLoopVideo: willLoopVideo
+            willLoopVideo: willLoopVideo,
+            setAudioSessionAmbient: setAudioSessionAmbient
         )
     }
 
-    /// Plays videos one after another on a UIView.
+    /// Plays a video.
     ///
     /// - Parameters:
     ///     - view: UIView that the video will be played on.
-    ///     - videoInfos: Array of `VideoInfo` for the videos to be played.
+    ///     - url: URL of the video. Can be from your local file system or the web.
     ///     - isMuted: Bool indicating whether video is muted. Defaults to true.
     ///     - alpha: CGFloat between 0 and 1. The higher the value, the darker the video. Defaults to 0.
     ///     - willLoopVideo: Bool indicating whether video should restart when finished. Defaults to true.
-    /// - Throws: `VideoBackgroundError.videoNotFound` if a video cannot be found.
+    ///     - setAudioSessionAmbient: Bool indicating whether to set the shared `AVAudioSession` to ambient. If this is
+    ///         not done, audio played from your app will pause other audio playing on the device. Defaults to true.
     public func play(view: UIView,
-                     videoInfos: [VideoInfo],
+                     url: URL,
                      isMuted: Bool = true,
                      alpha: CGFloat = 0,
-                     willLoopVideo: Bool = true) throws {
+                     willLoopVideo: Bool = true,
+                     setAudioSessionAmbient: Bool = true) {
         cleanUp()
 
-        guard !videoInfos.isEmpty else {
-            return
-        }
-
-        playerItems = try makeAVPlayerItems(videoInfos)
-
-        if playerItems.count > 1 {
-            if willLoopVideo {
-                playerItems.forEach { [weak self] in
-                    let observer = NotificationCenter.default.addObserver(
-                        forName: .AVPlayerItemDidPlayToEndTime,
-                        object: $0,
-                        queue: .main) { notification in
-                            if let currentItem = notification.object as? AVPlayerItem {
-                                currentItem.seek(to: kCMTimeZero, completionHandler: nil)
-                                self?.player.advanceToNextItem()
-                                self?.player.insert(currentItem, after: nil)
-                            }
-                    }
-                    self?.playerItemDidPlayToEndObservers.append(observer)
-                }
+        if setAudioSessionAmbient {
+            if #available(iOS 10.0, *) {
```

Comment:
Should we document somewhere that setAudioSessionAmbient is only available for iOS 10.0+?

Reply:
✅